### DISCUSSION TITLE
How to seek from Home Assistant?

### DISCUSSION THREAD

**Discussion by JoeMBlair** [music-assistant/support#2775] [Q&A] [answered]
I'm trying to seek a media player that was added by Music Assistant in Home Assistant. However it doesn't seem to support media player seek action, it's not in the entity dropdown. Is there another way to do it?

**Comment by OzGav** [comment]
Can you seek the original HA player? What player is it?

**Comment by JoeMBlair** [comment]
If I added it to home assistant separately then yea I can seek the player. However any added from player providers in music assistant that show up in home assistant don't have seek support. I can only seek them in music assistant and not home assistant.

**Reply by OzGav** [reply, marked as answer]
OK I see what you mean now. Transferring this to an issue.
